docker/buildkite/docker-compose-pinot.yml

```diff
@@ -0,0 +1,212 @@
+version: "3.5"
+
+services:
+  cassandra:
+    image: cassandra:4.1.1
+    ports:
+      - "9042:9042"
+    environment:
+      - "MAX_HEAP_SIZE=256M"
+      - "HEAP_NEWSIZE=128M"
+    networks:
+      services-network:
+        aliases:
+          - cassandra
+    healthcheck:
+      test: ["CMD", "cqlsh", "-u cassandra", "-p cassandra" ,"-e describe keyspaces"]
+      interval: 15s
+      timeout: 30s
+      retries: 10
+  prometheus:
+    image: prom/prometheus:latest
+    networks:
+      services-network:
+        aliases:
+          - prometheus
+    volumes:
+      - ./prometheus:/etc/prometheus
+    command:
+      - '--config.file=/etc/prometheus/prometheus.yml'
+    ports:
+      - '9090:9090'
+
+  kafka:
+    image: docker.io/bitnami/kafka:3.7
+    restart: unless-stopped
+    container_name: "kafka"
+    ports:
+      - "9092:9092"
+      - "9093:9093"
+    environment:
+      - KAFKA_CFG_ZOOKEEPER_CONNECT=zookeeper:2181
```

Comment:
Zookeeper version of bitnami/kafka

Reply:
nice! thanks for avoiding the wurstmeister image which has potential to break again in the future